integration-tests/rpc-get-protocol-constants.spec.ts

```diff
@@ -617,7 +617,7 @@ CONFIGS().forEach(({ lib, protocol, rpc }) => {
         hard_gas_limit_per_operation: new BigNumber(1040000),
         hard_gas_limit_per_block: new BigNumber(5200000),
         proof_of_work_threshold: new BigNumber(-1),
-        minimal_stake: "6000000000",
+        minimal_stake: '6000000000',
```

Comment:
Failure in the integration test, it is a bigNumber

Reply:
sorry must have been reverted when rebased